controllers/cloud.redhat.com/clowdapp_controller.go

```diff
@@ -100,6 +101,21 @@ type ClowdAppReconciler struct {
 // +kubebuilder:rbac:groups=networking.k8s.io,resources=ingresses;networkpolicies,verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups=config.openshift.io,resources=ingresses,verbs=get;list
 
+func (r *ClowdAppReconciler) reconcileMetricsStart() {
+	if !clowderconfig.LoadedConfig.Features.ReconciliationMetrics {
```

Comment:
Generally the positive here, especially as the function is so small, generates a better readability

```
if clowderconfig.LoadedConfig.Features.ReconciliationMetrics {
	r.ReconcileStartTime = time.Now()
}
```